go.mod

```diff
@@ -15,7 +15,7 @@ replace (
 
```

Comment:
Github doesn't allow me to comment there but the replace directive for metallb should be the following:
```
go.universe.tf/metallb => github.com/cilium/metallb v0.1.1-0.20210607221240-b4c60b959dd7
```